src/js/common/schemaform/FormIntroButtons.jsx

```diff
@@ -11,6 +11,8 @@ class FormIntroButtons extends React.Component {
     // Load the form data into the redux store if possible
     // NOTE: This does not fill the data into the form yet; just grabs it and
     //  saves it for later (when we want to fill in the blanks).
+    // TODO: Probably should wrap this in
+    // if (this.props.loggedIn) { ... }
```

Comment:
Let me know what you think about this. If they're not logged in, they'll get a 401 from the api, so I think we should go ahead with the change, but...I'd like an outside opinion.